x-pack/legacy/plugins/apm/public/components/app/ServiceMap/index.tsx

```diff
@@ -63,6 +71,9 @@ export function ServiceMap({ serviceName }: ServiceMapProps) {
     <div
       style={{ height: height - parseInt(theme.gutterTypes.gutterLarge, 10) }}
       ref={ref}
+      onMouseDown={() => {
```

Comment:
Instead of tracking upon `onMouseDown` should we do it when selecting a node?

https://github.com/elastic/kibana/blob/babefb5737ddd6cdd6b475dcf61d9d9788c8018a/x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Popover/index.tsx#L63-L65

Reply:
In discussion, it was decided to only track simple clicks inside the service map area, and increase granularity after beta. Do we want to revisit that, @nehaduggal?

Reply:
I'm not asking to track more, just trying to understand what we are currently tracking. If this is a service node click we are tracking, shouldn't we do it on the existing handler?

Reply:
We discussed this today, and are OK with higher granularity in the tracking. I'll make the updates. When we discussed earlier, there was only interest in clicks inside the area, regardless of which object was or wasn't clicked. The existing handlers will provide a better idea of _how_ the user interacts with it, rather than just a binary _interaction_ type event.